conftest.py

```
@@ -0,0 +1,33 @@
+
+def _patch_pydantic_url() -> None:
+    """Restore `pydantic.networks.Url` for dependencies expecting the legacy export."""
+    try:
+        networks = importlib.import_module("pydantic.networks")
+        if not hasattr(networks, "Url"):
+            from pydantic import AnyUrl
+
+            networks.Url = AnyUrl  # type: ignore[attr-defined]
+    except Exception:
+        # Best-effort compatibility; never block test collection.
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)